health-on-time/src/App.js

```diff
@@ -30,14 +30,14 @@ function App() {
   const [username, setUserName] = useState(null);
   const [email, setEmail] = useState(null);
   const [phone, setPhone] = useState(null);
-  const [partners, setPartners] = useState(null);
-  const [patients, setPatients] = useState(null);
+  const [partners, setPartners] = useState([]);
+  const [patients, setPatients] = useState([]);
   const [prescriptions, setPrescriptions] = useState(null);
   const [medications, setMedications] = useState(null);
 
   let history = useHistory();
-  // const BASE_URL = "http://localhost:3000/";
-  const BASE_URL = "https://health-on-time-api.herokuapp.com/";
+  const BASE_URL = "http://localhost:3000/";
```

Comment:
I'm guessing these can be (should be) changed back to heroku url 
![image](https://user-images.githubusercontent.com/71042305/111132581-267b9800-853f-11eb-80ac-836cb00c50fe.png)